solidity/test/merkle.test.ts

```diff
@@ -2,7 +2,7 @@ import { expect } from 'chai';
 import { utils } from 'ethers';
 
 import merkleTestCases from '../../vectors/merkle.json' assert { type: 'json' };
-import { TestMerkle, TestMerkle__factory } from '../types';
+import { TestMerkle, TestMerkle__factory } from '../core-utils/evm/types';
```

Comment:
pls update _all_ the references e.g. this

core-utils/evm -> core-utils/typechain

Reply:
All should be resolved